client/src/pages/Login/useStyles.ts

```diff
@@ -6,16 +6,23 @@ const useStyles = makeStyles(() => ({
     '& .MuiInput-underline:before': {
       borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
     },
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: '#fafafa',
   },
   authWrapper: {
-    minHeight: '100vh',
+    display: 'flex',
```

Comment:
All of these styles have already been applied to the `Box` component, you can remove them here

Reply:
Done the changes